### PR TITLE
Write --noproxy to .curlrc

### PR DESCRIPTION
**Pull request by mvidner**
After a PTF has been provided, this is for SLE11 SP4
https://bugzilla.suse.com/show_bug.cgi?id=923788
https://trello.com/c/dUtdKuNg/86-3-11-sp3-l3-923788-curlrc-is-missing-noproxy-setting

### CODE REVIEW

File: src/modules/Proxy.ycp [hunk 1]
```diff
@@ -99,6 +99,68 @@ global define boolean Read() {
     return true;
 }
 
+void WriteSysconfig() {
+    SCR::Write(.sysconfig.proxy.PROXY_ENABLED, enabled?"yes":"no");
```
Comment (jreidinger) [inline src/modules/Proxy.ycp]: please spaces arond `?:`

File: src/modules/Proxy.ycp [hunk 2]
```diff
@@ -99,6 +99,68 @@ global define boolean Read() {
...
+        proxyuser = user;
+        if (pass != "") proxyuser = user + ":" + pass;
+    }
+
+    map<string, string> options = $[
+        "--proxy-user": proxyuser,
+        // bnc#305163
+        "--proxy":      http,
+        // bsc#923788
+        "--noproxy":    no
```
Comment (jreidinger) [inline src/modules/Proxy.ycp]: not done in this pr, but variable with name `no` with file scope is quite horrible

**Comment by jreidinger** [comment]
few improvements but in general LGTM

**Comment by mchf** [comment]
Thanks for spliting Write* methods

File: src/modules/Proxy.ycp (hunk 2 above)
Comment (mvidner) [inline src/modules/Proxy.ycp]: Yes, I will rename it to `the`. Just kidding!

**Pull request merged**